advanced/is-pattern-2/templates/is/wso2is-pattern-2-identity-server-statefulset.yaml

```diff
@@ -0,0 +1,229 @@
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "is-pattern-2.resource.prefix" . }}-logstash-elasticsearch-creds
+                  key: username
+            - name: ELASTICSEARCH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "is-pattern-2.resource.prefix" . }}-logstash-elasticsearch-creds
+                  key: password
+            - name: ELASTICSEARCH_HOST
+              value: {{ .Values.wso2.centralizedLogging.logstash.elasticsearch.host }}
+        {{ end }}
+      serviceAccountName: {{ .Values.kubernetes.serviceAccount }}
+      {{- if .Values.wso2.deployment.wso2is.imagePullSecrets }}
+      imagePullSecrets:
+        - name: {{ .Values.wso2.deployment.wso2is.imagePullSecrets }}
+      {{- else if and (not (eq .Values.wso2.subscription.username "")) (not (eq .Values.wso2.subscription.password "")) }}
+      imagePullSecrets:
+        - name: {{ template "is-pattern-2.resource.prefix" . }}-wso2-private-registry-creds
+      {{ end }}
+      volumes:
+      - name: identity-server-conf
+        configMap:
+          name: {{ template "is-pattern-2.resource.prefix" . }}-identity-server-conf
+      - name: identity-server-bin
+        configMap:
+          name: {{ template "is-pattern-2.resource.prefix" . }}-identity-server-bin
+      - name: identity-server-conf-event-publishers
+        configMap:
+          name: {{ template "is-pattern-2.resource.prefix" . }}-identity-server-conf-event-publishers
+      - name: identity-server-conf-trust-store
+        configMap:
+          name: {{ template "is-pattern-2.resource.prefix" . }}-identity-server-conf-trust-store
+
+      {{ if .Values.wso2.deployment.persistentRuntimeArtifacts.sharedArtifacts.enabled }}
+      - name: identity-server-volume-claim-tenants-storage
+        persistentVolumeClaim:
+          claimName: {{ template "is-pattern-2.resource.prefix" . }}-identity-server-shared-tenants-volume-claim
+      - name: identity-server-volume-claim-userstores-storage
+        persistentVolumeClaim:
+          claimName: {{ template "is-pattern-2.resource.prefix" . }}-identity-server-shared-userstores-volume-claim
+      {{ end }}
+      {{ if .Values.wso2.centralizedLogging.enabled }}
+      - name: shared-logs
+        emptyDir: {}
+      - name: logstash-yml
+        configMap:
+          name: {{ template "is-pattern-2.resource.prefix" . }}-logstash-yml-conf
+      - name: c4-logstash-conf
+        configMap:
+          name: {{ template "is-pattern-2.resource.prefix" . }}-c4-logstash-conf
+      - name: shared-plugins
+        emptyDir: {}
+      {{ end }}
+      {{ if .Values.wso2.monitoring.enabled }}
+      - name: shared-prometheus-jmx-jar
+        emptyDir: {}
+      - name: identity-server-prometheus-exporter-conf
+        configMap:
+          name: {{ template "is-pattern-2.resource.prefix" . }}-identity-server-conf-prometheus-exporter
+      {{ end }}
```

Comment:
@darsham3 please add a new EOF line. Please check the rest of the files also and fix if an EOF line is missing.